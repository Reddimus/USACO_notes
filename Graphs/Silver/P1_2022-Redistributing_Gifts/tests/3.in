8
4 1 6 7 2 5 8 3
3 2 4 7 5 1 6 8
8 4 7 1 3 6 2 5
7 8 1 3 4 6 2 5
5 2 7 1 3 8 6 4
1 5 6 3 4 7 2 8
4 7 1 2 6 5 8 3
7 3 8 5 4 6 2 1
