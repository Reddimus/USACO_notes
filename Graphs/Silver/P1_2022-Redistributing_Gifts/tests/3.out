4
2
8
7
5
6
4
7
